frontend/src/modules/activity/components/activity-form-drawer.vue

```diff
@@ -328,7 +329,7 @@ const submit = () => {
       .then(() => {
         reset();
         emit('update:modelValue', false);
-        doFetch({});
+        fetchActivities(savedFilterBody.value);
```

Comment:
you can use this one as fetchActivities(null, true); its gonna take savedFilterBody